stories/topics/ref-aws-additional-configs-secure-controller-load-balancer.adoc

```diff
@@ -34,11 +34,12 @@ Ensure you are in the correct region.
 Ensure you are in the correct region.
 . Select the internal load balancer for {ControllerName}.
 . Click the *Listeners* tab.
-. Select the Listener, and then click btn:[Edit].
+. Select the Listener, and then click btn:[Add listener].
```

Comment:
Is there a choice of Listener? Is it obvious which to select?

Reply:
Here is the screenshot 
![image](https://github.com/ansible/aap-docs/assets/60555139/6c823fa0-9b0d-4715-8c9d-56b0c5c2747d)